packages/loaders/src/styled/StyledSkeleton.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { keyframes, css, ThemeProps, DefaultTheme } from 'styled-components';
+import rgba from 'polished/lib/color/rgba';
+import { DEFAULT_THEME, retrieveComponentStyles, getColor } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'loaders.skeleton';
+
+const fadeInAnimation = keyframes`
+  0%, 60% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+`;
+
+const skeletonAnimation = keyframes`
+  100% {
+    left: 100%;
+  }
+`;
+
+const skeletonRtlAnimation = keyframes`
+  100% {
+    right: 100%;
+  }
+`;
+
+const retrieveSkeletonBackgroundColor = ({
+  theme,
+  dark
+}: IStyledSkeletonProps & ThemeProps<DefaultTheme>) => {
+  if (dark) {
+    return css`
+      background-color: ${rgba(theme.colors.background, 0.2)};
+    `;
+  }
+
+  return css`
+    background-color: ${getColor('neutralHue', 800, theme, 0.1)};
+  `;
+};
+
+interface IStyledSkeletonProps {
+  width?: string;
+  height?: string;
+  dark?: boolean;
+  customWidth?: string;
+  customHeight?: string;
+}
+
+const retrieveSkeletonAnimation = ({ theme }: ThemeProps<DefaultTheme>) => {
+  if (theme.rtl) {
+    return css`
+      right: -1800px;
+      animation: ${skeletonRtlAnimation} 1.5s ease-in-out 300ms infinite;
+    `;
+  }
+
+  return css`
+    left: -1800px;
+    animation: ${skeletonAnimation} 1.5s ease-in-out 300ms infinite;
+  `;
+};
+
+const retrieveSkeletonGradient = ({
+  theme,
+  dark
+}: IStyledSkeletonProps & ThemeProps<DefaultTheme>) => {
+  // Disabling stylelint due to conflicts with prettier and linear-gradient formatting
+  return css`
+    /* stylelint-disable */
+    background-image: linear-gradient(
+      ${theme.rtl ? '-45deg' : '45deg'},
+      transparent,
+      ${dark ? getColor('kale', 700, theme, 0.4) : rgba(theme.colors.background, 0.6)},
```

Comment:
This should be using `getColor('chromeHue', ...)`.

Reply:
Oh neat. Didn't realize we had a `chromeHue`!